packages/lesswrong/components/recentDiscussion/RecentDiscussionTag.tsx

```diff
@@ -93,27 +59,21 @@ const styles = (theme: ThemeType): JssStyles => ({
 });
 
 const RecentDiscussionTag = ({ tag, refetch = () => {}, comments, expandAllThreads: initialExpandAllThreads, tagCommentType = "DISCUSSION", classes }: {
-  tag: TagRecentDiscussion | TagRecentSubforumComments,
+  tag: TagRecentDiscussion,
```

Comment:
The changes in this file are removing subforum-related code, plus some other code I saw was unused. I also deleted the `TagRecentSubforumComments` fragment since it wasn't used elsewhere.